.ruby-version

```diff
@@ -0,0 +1 @@
+2.5.3
```

Comment:
Should we be using `2.6.X` now?

Reply:
I took the version specified by [GitHub](https://pages.github.com/versions/).